.drone.yml

```diff
@@ -22,6 +22,17 @@ pipeline:
     commands:
       - tar -jxf owncloud-${SERVER_VERSION}.tar.bz2 -C /var/www
 
+  s3-app-download:
+    image: plugins/download:1
+    pull: true
+    source: https://github.com/owncloud/files_primary_s3/releases/download/v1.0.3/files_primary_s3-1.0.3.tar.gz
+
+  s3-app-unpack:
```

Comment:
You don't need to unpack or download it manually, just use something like this:

```
- OWNCLOUD_APPS_INSTALL=https://github.com/owncloud/files_primary_s3/releases/download/v1.0.3/files_primary_s3-1.0.3.zip
- OWNCLOUD_APPS_ENABLE=files_primary_s3
```

Reply:
THX for the hint - but via market it cannot be installed because of the version issue ...

![screenshot from 2019-01-22 07-51-27](https://user-images.githubusercontent.com/1005065/51517285-8e7c3b00-1e1a-11e9-8ae0-49c1140d502f.png)